components/Footer.module.scss

```diff
@@ -0,0 +1,145 @@
+.footer{ 
+    background: var(--main-page-background-dark-mode);
```

Comment:
formatting is wrong, make sure you're using `.prettier` and have that installed on your computer